### DISCUSSION TITLE
Vercel build "unexpected internal error"

### DISCUSSION THREAD

**Discussion by TheNetAdmin** [vercel/community#436] [Help] [answered]
Deploying a site using Gatsby and suddenly getting the error `An unexpected internal error occurred` after the build is finished. Attached is the end of the log, which shows Gatsby is done building, while some internal error occurred when deploying.

```
info Done building in 38.700970501 sec
Done in 40.89s.
Uploading build outputs...
Deploying build outputs...
An unexpected internal error occurred
```

I then tried to re-deploy a previously deployed version, the same unexpected internal error, while ~2hrs ago it was a success, with the exact same code.

Is there any way to debug this?

**Comment by TheNetAdmin** [comment, marked as answer]
Seems ok now, no such error anymore.

**Comment by talatkuyuk** [comment]
Today, I faced exactly the same problem. It is Nextjs application.
```
Traced Next.js server files in: 320.236ms
Created all serverless functions in: 1.163s
Deploying build outputs...
An unexpected internal error occurred
```

I tried 2-3 times, redeploy, etc, the same problem occurs

**Comment by meerface** [comment]
Hello, we just hit this exact issue too. Anyone have any idea?

**Comment by subzero79** [comment]
Same here build apparently succeds , but fails on upload maybe?

```
Done in 121.64s.
--
15:43:03.255 | Traced Next.js server files in: 414.918ms
15:43:13.841 | Created all serverless functions in: 10.586s
15:43:26.996 | Uploading build outputs...
15:43:29.949 | An unexpected internal error occurred
```

**Reply by tanomsakk** [reply]
Exactly the same, build success and fail when uploading

**Comment by jmarioste** [comment]
Same here. seems to be a problem with vercel

**Comment by swarnava** [comment]
Hi everyone, We are currently investigating this and update you shortly. Apologize for the inconvenience. Meanwhile, if you can create a Support case, we can notify you once this is resolved:  https://vercel.com/support/request

**Comment by bifard** [comment]
Same here. seems to be a problem with vercel

**Comment by nicogulo** [comment]
did everyone get the same error in the last 3 hours?

**Comment by swarnava** [comment]
Can you redeploy and let us know if you are still facing issue?

**Comment by bifard** [comment]
All ok!) 
--
Отправлено из Mail.ru для Android четверг, 21 апреля 2022г., 15:23 +07:00 от Swarnava Sengupta  ***@***.*** :

**Comment by subzero79** [comment]
All good now :+1: !!

**Comment by radikris** [comment]
Today I got the same error, anyone else? (after 30 min building)
![image](https://user-images.githubusercontent.com/55052220/166940218-762d2c2f-b4b2-4358-ae14-e8ad3edff9e2.png)

**Comment by ifeanyiisitor** [comment]
Experiencing this issue right now

**Reply by leerob** [reply]
Sorry about this - could you open a support request so we can investigate? 

https://vercel.com/support/request

**Reply by SakarDev** [reply]
Same issue at verification by phone number! 
"Error: An unexpected internal error occurred"

**Reply by amyegan** [reply]
Hi @SakarDev! Are you getting this error when signing up for a new account or attempting to log in? If so, please contact Vercel Support by emailing [registration@vercel.com](mailto:registration@vercel.com) with the email address you used to sign up.

If this is an error happening during phone verification in your app, that sounds like a slightly different scenario from what was being discussed here. In that case, please [create a new discussion](https://github.com/vercel/community/discussions/new) with some information about what you're seeing and a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can help figure out what is going wrong.

**Comment by Sirk** [comment]
Hello Vercel team ! 
I experience the same problem today, with the same files/build than yesterday which was okay. 
Is this a Vercel issue ? 
<img width="610" alt="image" src="https://user-images.githubusercontent.com/1640743/186381665-ee36e5a5-104c-4cde-8e47-28052de23142.png">

**Reply by samuelhorn** [reply]
@Sirk Same here!

**Reply by swarnava** [reply]
Are you deploying edge function? We are having a known issue https://www.vercel-status.com/incidents/4ff3h89mx9hk

**Reply by samuelhorn** [reply]
@swarnava working again! :)

**Reply by Sirk** [reply]
@swarnava seems like it's fixed ! big up to the team :)

**Comment by filipbarak** [comment]
I am deploying a simple static page and am getting this error as well:


λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
--
23:04:56.247 | ○  (Static)  automatically rendered as static HTML (uses no initial props)
23:04:56.247 |  
23:04:57.088 | Traced Next.js server files in: 702.375ms
23:04:57.461 | Created all serverless functions in: 372.523ms
23:04:57.478 | Collected static files (public/, static/, .next/static): 12.956ms
23:04:57.990 | Build Completed in /vercel/output [26s]
23:05:01.842 | An unexpected internal error occurred

**Reply by swarnava** [reply]
Please create a separate thread for this along with deployment url!
